packages/webapp/components/TreasuryBalance.tsx

```diff
@@ -19,12 +19,40 @@ import { useGnarsV2TokenBalanceOf } from "utils/sdk"
 import { formatEther } from "viem"
 import { useBalance } from "wagmi"
 import { ShredIcon } from "./Icons"
-
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { set } from "lodash"
+import { abbreviatedBalance } from "utils/numberAbreviation"
 export const TreasuryBalance = () => {
   const { data: treasuryBalance } = useBalance({
     address: TREASURY_ADDRESS,
   })
 
+  const [usdcBalance, setUsdcBalance] = useState(0);
+
+
+  useEffect(() => {
+    const fetchUSDCBalance = async () => {
+      try {
+        const address = "0xa1B74d2280966A89AC7e0F3A8bc5f0867C776d98";
```

Comment:
might be nice to have something like this in a separate "constants" file for reuse in other places and so the variable can be named something like `GNARS_DAO_MULTISIG`

Reply:
we have, I just saw we import it to the file already, I am replacing it right now

<img width="1151" alt="image" src="https://github.com/gnars-dao/gnars-frontend/assets/116202536/a3c9e135-0e82-41d8-b4bb-476e64c3b7b9">